saleor/plugins/webhook/plugin.py

```diff
@@ -915,14 +915,23 @@ def draft_order_deleted(self, order: "Order", previous_value: Any) -> Any:
                 order_data, event_type, webhooks, order, self.requestor
             )
 
-    def fulfillment_created(self, fulfillment: "Fulfillment", previous_value):
+    def fulfillment_created(
+        self,
+        fulfillment: "Fulfillment",
+        notify_customer: Optional[bool] = True,
+        previous_value: Optional[Any] = None,
+    ):
         if not self.active:
             return previous_value
         event_type = WebhookEventAsyncType.FULFILLMENT_CREATED
         if webhooks := get_webhooks_for_event(event_type):
             fulfillment_data = generate_fulfillment_payload(fulfillment, self.requestor)
             trigger_webhooks_async(
-                fulfillment_data, event_type, webhooks, fulfillment, self.requestor
+                fulfillment_data,
+                event_type,
+                webhooks,
+                {"notify_customer": notify_customer},
```

Comment:
It won't work.
```suggestion
                {
                   "notify_customer": notify_customer,
                   "fulfillment": fulfillment,
                   "order": fulfillment.order,
                },
```

Reply:
Even suggest to pass:
{
   "notify_customer": notify_customer,
   "fulfillment": fulfillment,
}
as order can be resolved inside subscription type